keyboards/ambergon/multipleglove/keymaps/default/keymap.c

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
```

Comment:
This does not look representative of the physical layout:
![image](https://github.com/user-attachments/assets/102ab901-5ba1-4ed1-ba28-a72d7558b3a6)